src/utils/formHelperFunctions.ts

```diff
@@ -820,5 +826,14 @@ export function httpHelpers(client: any) {
         headers: Record<string, string>
       ) => client.runCustomRequest(method, url, data, headers))
   );
+
+  helpers.connectToAPI = async (name: string) => {
```

Comment:
should this actually just be integrated into the runCustomRequest interface? maybe the url parameter can be polymorphic and also support `name`? This way we have just one interface and users can still pass in custom data and headers to adjust the api connector if they want

Reply:
Yeah, I can do that -- ideally we move completely away from the older payload all together in the future.

Reply:
i think we were going to rename this to connect as well